### PR TITLE
Add mypy to backend cli

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

File: parsec/backend/cli/sequester.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-from base64 import b64encode, b64decode
-from async_generator import asynccontextmanager
-import textwrap
 import attr
 import click
-from parsec._parsec import DateTime
-from typing import Optional, Dict, List, Tuple
+import oscrypto
+import textwrap
+from oscrypto.asymmetric import PrivateKey
```
Comment (FirelightFlagboy) [inline parsec/backend/cli/sequester.py:8]: Do we plan to replace `oscrypto` by the Rust impl ?

**Review by FirelightFlagboy — APPROVED** [review]

File: parsec/backend/cli/sequester.py (hunk 1 above)
Comment (FirelightFlagboy) [reply parsec/backend/cli/sequester.py:8]: @TimeEngineer up

**Review by FirelightFlagboy — APPROVED** [review]

File: parsec/backend/cli/sequester.py (hunk 1 above)
Comment (TimeEngineer) [reply parsec/backend/cli/sequester.py:8]: Do you think we can just replace ?

Comment (FirelightFlagboy) [reply parsec/backend/cli/sequester.py:8]: I don't know that wasn't the question :shrug: 
More like if we want to restrain the usage of crypo to the crate `libparsec-crypto`

Comment (TimeEngineer) [reply parsec/backend/cli/sequester.py:8]: We should

Comment (FirelightFlagboy) [reply parsec/backend/cli/sequester.py:8]: I've created an issue for that #3327

**Pull request merged**